pkg/featureSet/persistentImages/persistentImages.go

```diff
@@ -0,0 +1,45 @@
+package persistentImages
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/openshift/ocm-container/pkg/engine"
+	"github.com/openshift/ocm-container/pkg/ocm"
+)
+
+const (
+	destDir   = "/var/lib/containers/"
+	sourceDir = "/.cache/ocm-container/images/"
+)
+
+type Config struct {
+	Env    map[string]string
+	Mounts []engine.VolumeMount
+}
+
+func New(home string) (*Config, error) {
+	var err error
+
+	config := &Config{}
+
+	ocmClient, err := ocm.NewClient()
+	if err != nil {
+		return config, err
+	}
+	defer ocmClient.Close()
+
+	mount := filepath.Join(home, sourceDir)
```

Comment:
```suggestion
	log.Debug("using image cache mount: " + mount)
	mount := filepath.Join(home, sourceDir)
```

Not a deal breaker at all - just suggesting some debug logging for troubleshooting.